tools/releaseBuild/Images/microsoft_powershell_windowsservercore/DockerFile

```diff
@@ -1,10 +1,10 @@
 # escape=`
 #0.3.6 (no powershell 6)
-FROM microsoft/windowsservercore
+FROM microsoft/powershell:windowsservercore
 LABEL maintainer='PowerShell Team <powershellteam@hotmail.com>'
 LABEL description="This Dockerfile for Windows Server Core with git installed via chocolatey."
 
-SHELL ["C:\\Windows\\System32\\WindowsPowerShell\\v1.0\\powershell.exe", "-command"]
+SHELL ["C:\\Program Files\\PowerShell\\latest\\pwsh.exe", "-command"]
```

Comment:
I guess the `install-powershell.ps1` part below should be removed now?

Reply:
done